vms/avm/wallet_service.go

```diff
@@ -302,7 +306,7 @@ func (w *WalletService) SendMultiple(_ *http.Request, args *SendMultipleArgs, re
 		return err
 	}
 
-	txID, err := w.issue(tx.Bytes())
+	txID, err := w.issue(tx)
```

Comment:
We need to ensure atomic access between `update` and `issue` - so we use the context lock for exclusion throughout this API.